.cproject

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
i guess this file also needs to be removed, since .gitignore filters this one out.

Reply:
fixed